src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getPageNumbers } from '../../utils';
+
+type PaginationProps = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<PaginationProps> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages: number[] = getPageNumbers(total, perPage);
```

Comment:
You could get the result of this function on the higher component, and here just receive totalPages prop

Reply:
I've thought in this solution, but at ReadMe file, they ask the specifics props, so I used this way.